serverless/cmd/keys/main.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Maybe rename this directory to `generate_key` or similar?
The `go build` will create a binary called `keys` otherwise.